### PR TITLE
Feature/variables

### PR DESCRIPTION
**Pull request by iruzevic**
- the same as on frontend libs

### CODE REVIEW

File: src/Blocks/AbstractBlocks.php [hunk 1]
```diff
@@ -359,7 +359,7 @@ protected function getComponents(): array
 			throw InvalidBlock::missingComponentsException();
 		}
 
-		$components = array_diff((array)$paths, ['..', '.']);
+		$components = array_diff((array)$paths, ['..', '.', '.DS_Store']);
```
Comment (dingo-d) [inline src/Blocks/AbstractBlocks.php:362]: Why is this added? This is mac specific, and shouldn't be in the code.

Comment (iruzevic) [reply src/Blocks/AbstractBlocks.php:362]: I have no idea for some reason 15 test started to fail for no apparent reason, I tried everything.

**Review by dingo-d — APPROVED** [review]

**Review by dingo-d — CHANGES_REQUESTED** [review]
DS_Store should be removed.

File: src/Blocks/AbstractBlocks.php (hunk 1 above)
Comment (dingo-d) [reply src/Blocks/AbstractBlocks.php:362]: Because you had `.DS_Store`? That makes no sense.

Comment (iruzevic) [reply src/Blocks/AbstractBlocks.php]: > DS_Store should be removed.

Done. I have removed all . files from the scan

File: tests/Helpers/ComponentHelpersTest.php [hunk 2]
```diff
@@ -402,377 +389,260 @@ function ($argument) {
...
-	$attributes = [
-		'testSelectVariable' => "test-3",
-	];
-
-	$output = Components::outputCssVariables(
-		$attributes,
-		$manifest,
-		'uniqueString'
-	);
+	var_dump($output);
```
Comment (IvanGrginovInf) [inline tests/Helpers/ComponentHelpersTest.php:573]: Remove var_dump

**Review by IvanGrginovInf — CHANGES_REQUESTED** [review]
Single comment regarding var_dump, otherwise looks good.

**Review by dingo-d — APPROVED** [review]

File: tests/Helpers/ComponentHelpersTest.php (hunk 2 above)
Comment (iruzevic) [reply tests/Helpers/ComponentHelpersTest.php]: fixed
